### PR TITLE
(PE-8657) Puppet module for AIX mount provider fix.

### PR DESCRIPTION
**Pull request by geoffnichols**
This is a simple Puppet module to replace two files
in the mount type/provider. This is the proposed hotfix 
delivery vehicle for https://github.com/puppetlabs/puppet/pull/3769.

### CODE REVIEW

**Comment by geoffnichols** [comment]
The README.md and manifests/init.pp are probably the files that need critique here.

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+       d. Click 'Add rule'.
+  7. Select the Matching nodes tab and confirm the list of nodes is correct.
+  8. On the 'Matching nodes' tab, inspect the matching nodes to ensure they are correct.
+  9. Commit the changes (link is to the bottom right of the screen).
+  10. On the 'Classes' tab, select 'aix_mount_fix' and click 'Add class'.
+  11. Commit the changes (link is to the bottom right of the screen).
+
+###How to confirm resolution
+
+1. Run `/opt/puppetpuppet agent -t` on an AIX agent or wait for the service to have run automatically.
```
Comment (hunner) [inline README.md]: /opt/puppet/bin/puppet

Comment (geoffnichols) [inline README.md]: This is updated now.

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+  10. Commit the change (link is to the bottom right of the screen).
+
+  * To apply the fix to all AIX PE 3.7 agent nodes running PE 3.7:
+
+  1. Log into the console.
+  2. Go to the Classification tab in the console.
+  3. If the 'PE AIX 3.7' node group hasn't been created, set a value of 'PE AIX 3.7' for 'Node group name' and click 'Add group'.
+  4. Click on the 'PE AIX 3.7' group link.
+  5. On the Rules tab:
+       a. Select 'puppetversion' from the Fact pulldown.
```
Comment (jpadams) [inline README.md]: 5. ensure that the selector above the rules section reads "Nodes in this node group match [ All ]  of the following rules:"

**Comment by stahnma** [comment]
LGTM

**Comment by geoffnichols** [comment]
Updated to fix module uninstall command.

**Comment by shrug** [comment]
:+1:

**Comment by melissa** [comment]
:+1: I don't have the ability to merge this, otherwise I would...

**Pull request merged**